addons/mail/static/src/webclient/commands/mail_providers.js

```diff
@@ -66,3 +66,10 @@ commandProviderRegistry.add("channel", {
         }));
     },
 });
+
+const commandFooterRegistry = registry.category("command_footer");
```

Comment:
this could be done in a single line maybe?

Reply:
We will probably add few tips